src/util/countryDataManager.ts

```diff
@@ -0,0 +1,190 @@
+import fetch from "node-fetch";
+import { logger } from "../logger";
+
+export interface Country {
+	name: string;
+	official_name: string;
+	cca2: string;
+	tld: string[];
+	unMember: boolean;
+	population: number;
+	capital: string[];
+	languages: string[];
+	currencies: string[];
+	timezones: string[];
+	region: string;
+	subregion: string;
+	latitude: number;
+	longitude: number;
+	area: number;
+	maps: { googleMaps: string; openStreetMaps: string };
+	flags: { png: string; svg: string };
+}
+
+export let countryData: Country[] = [];
```

Comment:
why and what does that even mean?

Reply:
exporting a let, means that other functions are able to modify data. "mutable" aka "modifyable".
you should export a getter function instead, which accesses the private field, which might be mutable, so that noone else is able to modify it

Reply:
Do you mean like this?
```suggestion
let countryData: Country[] = [];

export function getCountryData (): Country[] {
      return countryData;
}
```

Reply:
Technicly invalid issue, the variable will be mutable even if it was a const and exported with a getter because the variable itself is just a reference which contents can be modifed unless frozen.